rockcraft/models/project.py

```diff
@@ -337,7 +339,12 @@ def _validate_entrypoint_service(
                 "a valid Pebble service."
             )
 
-        command = values.get("services")[entrypoint_service].command
+        services = values.get("services")
```

Comment:
Probably clearer to do `command = values["services"][entrypoint_service].command`, because the check just before this code already ensures that the dict entries exist